apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/OptionalAssertionsSpec.kt

```diff
@@ -1,13 +1,13 @@
 package ch.tutteli.atrium.api.fluent.en_GB.jdk8
 
 import ch.tutteli.atrium.creating.Expect
-import ch.tutteli.atrium.specs.fun0
+import ch.tutteli.atrium.specs.*
 import ch.tutteli.atrium.specs.integration.OptionalAssertionsSpec
-import ch.tutteli.atrium.specs.notImplemented
 import java.util.*
 
 class OptionalAssertionsSpec : OptionalAssertionsSpec(
-    isEmpty = fun0(Expect<Optional<Any>>::isEmpty)
+    isEmpty = fun0(Expect<Optional<Any>>::isEmpty),
+    isPresent =fun1<Optional<Any>, Any>(Expect<Optional<Any>>::isPresent)
```

Comment:
I suggest you use `Int` instead of `Any` so that you can make assertions about an `Int` in your specs